framework/include/base/Assembly.h

```diff
@@ -768,87 +768,78 @@ class Assembly
   void copyNeighborShapes(MooseVariableField<T> & v);
   void copyNeighborShapes(unsigned int var);
 
+  /**
+   * Key structure for APIs manipulating global vectors/matrices
+   */
```

Comment:
please add a one-liner on how to use this.
You have to imagine the dev in the "blessed class" header seeing this being used, right clicking in VSCode and jumping to this definition

Reply:
I will imagine the dev loading their emacs tags table and jumping here 😆